components/eamxx/src/share/io/scorpio_output.cpp

```diff
@@ -89,8 +102,13 @@ AtmosphereOutput (const ekat::Comm& comm, const ekat::ParameterList& params,
 {
   using vos_t = std::vector<std::string>;
 
-  if (params.isParameter("Fill Value")) {
-    m_fill_value = static_cast<float>(params.get<double>("Fill Value"));
+  if (params.isParameter("fill_value")) {
+    m_fill_value = static_cast<float>(params.get<double>("fill_value"));
```

Comment:
Is there a default for `m_fill_value`?

Reply:
yes, it is assigned in the `.hpp` file: https://github.com/E3SM-Project/scream/pull/2433/files#diff-47d669cb9ede18b477ba793844458e7c86ec56590372ce145d17412eaed0d742R202